pkg/cmd/bazci/githubpost/issues/formatter_unit.go

```diff
@@ -155,7 +155,7 @@ func (unitTestFormatterTyp) Body(r *Renderer, data TemplateData) error {
 		r.A("Improve this report!",
 			"https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues",
 		)
-		r.Escaped("\n")
+		r.Escaped("\n\n") // need a blank line here since we have one above
```

Comment:
Interesting, I see all our issues have the Jira issue displaying as part of the sub! Nice catch!